src/plone/app/workflowmanager/browser/layout.py

```diff
@@ -0,0 +1,55 @@
+from Products.Five.browser import BrowserView
+
+import json
+from plone import api
+
+
+class GraphLayout(BrowserView):
+    """Class to handle the Workflow Manager graph layouts.
+
+        propSheet:
+                the property sheet containing all of the layouts
+
+        layout:
+                the actual value of the individual property on the propSheet
+                representing the layout of the graph
+    """
+
+    def __init__(self, context, request, workflow=None):
+        self.context = context
+        self.request = request
+
+        self.REGISTRY_KEY = "plone.app.workflowmanager.layouts"
```

Comment:
you should move this code out of `__init__()` to a `setup()` method to avoid hiding issues:

https://docs.plone.org/develop/plone/views/browserviews.html#view-init-method-special-cases